src/pages/Event/Event2023FallMissions.tsx

```diff
@@ -7,10 +11,22 @@ import WhiteContainer from "components/WhiteContainer";
 
 import theme from "tools/theme";
 
-const MissionContainer = () => {
+import { ReactComponent as CreditIcon } from "static/events/2023fallCredit.svg";
+
+type MissionContainerProps = {
+  quest?: Quest;
+};
+const MissionContainer = ({ quest }: MissionContainerProps) => {
+  const { completedQuests } = useValueRecoilState("event2023FallInfo") || {};
+  const [isDone, questCompletedCnt] = useMemo(() => {
+    const cnt =
+      completedQuests?.filter((questId) => questId === quest?.id).length || 0;
+    const isDone = cnt && quest?.maxCount ? cnt >= quest?.maxCount : false;
```

Comment:
`cnt` 는 무조건 `type` 이 `number` 입니다.
따라서 `isDone` 에서 `cnt`의 존재 여부를 확인할 필요가 없습니다. (cnt가 0일 때, `if (cnt)` 결과 값이 `false` 이긴 하지만)

`isDone` 계산 더 간단히 할 수 있을 것 같아요.

```suggestion
    const cnt =
      completedQuests?.filter((questId) => questId === quest.id).length || 0;
    const isDone = quest.maxCount ? cnt;

```